index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name=" viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Skeleton</title>
+</head>
+<body>
+  <div id="mobile-background">
+    
+    
+    <header>
+      <h1>Inyambo</h1>  
+    
+      
+      <nav>
+        <div id="Menu-bar"></div>
+      </nav>
+    </header>
+    
+    <section id="content">
+      <h2>I’m Inyambo <br> Glad to see <br> you!</h2>
+      <p>
+        I am a software developer! I can help you build a product , 
+        feature or website. You can Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, please
+         don't hesitate to contact me.
+      </p>
+      <h3>LET'S CONNECT</h3>
+      <a href="#">
+       <ul id="Social">
+         <li class="icon-image"></li>
```

Comment:
-[ ] Kindly wrap each social icon in a tag 
<img width="1418" alt="Screen Shot 2022-08-17 at 03 18 39" src="https://user-images.githubusercontent.com/81444338/185095575-c10b0716-b6d6-4f04-8a17-36a47dc398af.png">